src/modules/keycloak-administration/domain/keycloak-user.service.ts

```diff
@@ -163,6 +163,56 @@ export class KeycloakUserService {
         }
     }
 
+    public async updateUser(
+        username: string,
+        oxUserName: OXUserName,
+        oxContextName: OXContextName,
+    ): Promise<Result<void, DomainError>> {
```

Comment:
It this is just used to update OX-related details, it should probably have a more specific name